samples/bookinfo/routing-rules/destination-policy-reviews.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews-destination
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
```

Comment:
This is also optional isn't it?

Reply:
Actually ROUND_ROBIN is the documented default: https://github.com/istio/api/blob/ac6b6fef5693dce7c2cda71a18aefb4c07fb6af9/networking/v1alpha3/destination_rule.proto#L195

I'm not exactly sure where this yaml file is actually used, but I think it's meant to be an example of how to change the loadbalancer policy for a service.